.github/workflows/unit-test.yml

```
@@ -82,9 +82,26 @@ jobs:
           php-version: ${{ matrix.php }}
           github-token: ''
           extensions: :xdebug, redis
+          ini-values: opcache.enable=0, opcache.enable_cli=0
       - name: Initialize Composer
         uses: ./.github/actions/composer
```

actionlint (1.7.12)

[error] description is required in metadata of "" action at "/home/jailuser/git/.github/actions/composer/action.yml"

(action)